dashboard/lib/sign_up_tracking.rb

```diff
@@ -1,11 +1,38 @@
 require 'cdo/firehose'
+require 'dynamic_config/dcdo'
 
 module SignUpTracking
-  def self.begin_sign_up_tracking(session)
+  NOT_IN_STUDY_GROUP = 'v1'
+  CONTROL_GROUP = 'v2-control'
+  NEW_SIGN_UP_GROUP = 'v2-finish-sign-up'
```

Comment:
We've effectively got three groups:

1. `v2-control` - Google and Clever OAuth sign-ups seeing the old experience.
1. `v2-finish-sign-up` - Google and Clever OAuth sign-ups seeing the new experience.
1. `v1` - All other sign-ups.